atc/db/team.go

```diff
@@ -1331,18 +1331,20 @@ func (t *team) insertJobPipes(tx Tx, jobConfigs atc.JobConfigs, resourceNameToID
 	}
 
 	for _, jobConfig := range jobConfigs {
-		for _, plan := range jobConfig.Plans() {
+		var err error
+		jobConfig.PlanConfig().Each(func(plan atc.PlanConfig) {
+			if err != nil {
+				return
+			}
+
 			if plan.Get != "" {
 				err = insertJobInput(tx, plan, jobConfig.Name, resourceNameToID, jobNameToID)
```

Comment:
Maybe this should only overwrite err if the returned err is not nil? Otherwise this might clobber the earlier error?

Reply:
oh wait, I missed the err != nil at the beginning